modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,694 @@
+
+The **Premium Skins and Icon Packs** were rebuilt to pull in the changes also incorporated into the default {productname} 6.3 skin, Oxide.
+
+For information on using premium skins and icon packs, see: xref:premium-skins-and-icons.adoc[Premium Skins and Icon Packs].
+
+[[improvements]]
+== Improvements
+
+{productname} 6.3 also includes the following improvements
+
+=== The Text color and Background color menus now display a check mark denoting the currently selected color
+
+Previously, the color swatches displayed when the Text color and Background color menus were opened gave no indication as to whether or not the Text color or Background color of the current selection was one of the colors displayed in these menus.
+
+With this update, when the selected text’s current Text color or Background color matches one of the colors available from the appropriate color swatch menu, a check mark (the ✔️ character, also known as a tick) displays over that color.
+
+=== Text color and Background color toolbar menus now pre-select the color for the selected text or insertion point position.
+
+{productname} 6.3 included an improvement to the Text color or Background color menus that allows the user to identify the pre-populated color for the selected-text: a ✔️ character displayed over the color that matches the currently selected Text color or Background color.
+
+In addition to this, in {productname} 6.3, `getCurrentColor` checks for the first found `backcolor` and `forecolor` color before the parent color, based on the current selection or insertion point position.
+
+As a consequence, when either the Text color or Background color toolbar menus are opened, if the selected Text color or Background color matches one of the colors available from the appropriate color swatch menu, that color is pre-selected.
+
+
+[[additions]]
+== Additions
+
+{productname} 6.3 also includes the following additions:
+
+=== New expand functions added to tinymce.selection and tinymce.dom.RangeUtils which expand the selection around the nearest word.
+
+{productname} 6.3 introduced two new API functions that allow a user to select a word from the insertion point location which applies a new expand feature.
+
+This new `tinymce.selection` API function allows {productname} users and developers to expand the selection range to contain the entire word when the selection is collapsed within the word.
+
+To support this new API function, {productname} 6.3 also introduced `tinymce.dom.RangeUtils`, which focuses on a new `expand` method range that expands around the entire word that the user selection was collapsed within by the `tinymce.selection` API function.
+
+If, in the following example, a user places the insertion point between the first and second `m` in the word, `comment`, then clicks the _Add comment_ toolbar button, or selects _Insert > Add comment_ from the TinyMCE menu, or presses the _Add comment_ keyboard shortcut (eg ⌘+⌥+M) — 
```

Comment:
```suggestion
These new API functions are implemented to improve the selection of whole words in the Comments plugin. If, in the following example, a user places the insertion point between the first and second `m` in the word, `comment`, then clicks the _Add comment_ toolbar button, or selects _Insert > Add comment_ from the TinyMCE menu, or presses the _Add comment_ keyboard shortcut (eg ⌘+⌥+M) — 
```
Missing some relation to the comments plugin and below example, since the example isn't of the use of these APIs, but rather an improvement made using them.